### PR TITLE
try ci

### PR DESCRIPTION
**Pull request by toxa81**

### CODE REVIEW

**Comment by toxa81** [comment]
cscs-ci run default

File: .ci/cscs.yml [hunk 1]
```diff
@@ -11,5 +11,35 @@ build image:
...
+.run_tests:
+  extends: .container-runner-daint-gpu
+  needs: ["build image"]
+  stage: test
+  image: $CSCS_REGISTRY_PATH/q-e-sirius/q-e-sirius-ci:$CI_COMMIT_SHA
+  variables:
+    CRAY_CUDA_MPS: 1
+    GIT_STRATEGY: none
+    MPICH_MAX_THREAD_SAFETY: multiple
+    CSCS_REGISTRY_LOGIN: 'YES'
```
Comment (finkandreas) [inline .ci/cscs.yml]: Remove CSCS_REGISTRY_LOGIN, this is deprecated and unused.

File: .ci/cscs.yml [hunk 2]
```diff
@@ -11,5 +11,35 @@ build image:
...
+    CSCS_REBUILD_POLICY: always
+
+.run_tests:
+  extends: .container-runner-daint-gpu
+  needs: ["build image"]
+  stage: test
+  image: $CSCS_REGISTRY_PATH/q-e-sirius/q-e-sirius-ci:$CI_COMMIT_SHA
+  variables:
+    CRAY_CUDA_MPS: 1
+    GIT_STRATEGY: none
```
Comment (finkandreas) [inline .ci/cscs.yml]: `GIT_STRATEGY: none` This is the default value anyway, could be deleted.

File: .ci/cscs.yml [hunk 3]
```diff
@@ -11,5 +11,35 @@ build image:
...
+    MPICH_MAX_THREAD_SAFETY: multiple
+    CSCS_REGISTRY_LOGIN: 'YES'
+    PULL_IMAGE: 'YES'
```
Comment (finkandreas) [inline .ci/cscs.yml]: `PULL_IMAGE: 'YES'` This is the default, could be deleted.

File: .ci/cscs.yml [hunk 4]
```diff
@@ -11,5 +11,35 @@ build image:
...
+    SLURM_HINT: nomultithread
+    SLURM_UNBUFFEREDIO: ''
```
Comment (finkandreas) [inline .ci/cscs.yml]: not super useful, besides that you get thousands of almost empty lines while the image is being pulled

**Review by finkandreas — COMMENTED** [review]
A couple of values are the default anyway, and could be removed (but if you want to keep them for clarity, it is also ok to keep them).

File: .ci/cscs.yml (hunk 4 above)
Comment (toxa81) [reply .ci/cscs.yml]: I removed SLURM_UNBUFFEREDIO completely

**Comment by toxa81** [comment]
@gcistaro should be `test-suite`, not `test_suite`

**Pull request closed without merging**